### ISSUE TITLE
gelu in mlp_mixer is not called

### ISSUE THREAD

**Issue by scott306lr** [leondgarse/keras_mlp#2]
As said in the title,  gelu in mlp_mixer is actually not called.
The 11th line of mlp_mixer.py made `activation = None`:
```py
def mixer_block(inputs, tokens_mlp_dim, channels_mlp_dim=None, activation=None, name=None):
```
which should actually be:
```py
def mixer_block(inputs, tokens_mlp_dim, channels_mlp_dim=None, activation="gelu", name=None):
```

I found it out because the model isn't converging well on CIFAR10, even with pretrained weights from [google-research/vision_transformer](https://github.com/google-research/vision_transformer) loaded.

Everything worked fine then after rewriting `activation="gelu"`.

**Comment by leondgarse** [comment]
I have updated the fullly architecture of `mlp_mixer`, and uploaded the pre-trained weights reloaded from [Github google-research/vision_transformer](https://github.com/google-research/vision_transformer#available-mixer-models).
Model forward related changes are:
- Add a parameter `activation` on model definition level, which controls all `activation` in the whole model. Default as `"gelu"`.
- `axis` in `LayerNormalization` actually should be `-1` for keras `"channels_last"` one.

**Comment by scott306lr** [comment]
Thanks for your update, helped a lot!   New function is very useful.
